### PR TITLE
Ibtida genesis config

### PR DESCRIPTION
**Pull request by justinfrevert**
Adds chain spec for testnet and gives tokens to faucet account in genesis block.
Specifies an address prefix of 92.
Updates token decimal count.

### CODE REVIEW

File: runtime/src/constants.rs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pub mod tokens {
+	pub const TOKEN_COUNT: u128 = 5_000_000_000 * 1_000_000_000;
```
Comment (nblogist) [inline runtime/src/constants.rs]: @justinFrevert 
>1_000_000_000

this should be with 18 0s if it is the accuracy of anmol

**Review by nblogist — DISMISSED** [review]

**Review by aelesbao — DISMISSED** [review]

**Review by nblogist — DISMISSED** [review]

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -41,7 +41,7 @@ pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
 pub fn chain_properties() -> Properties {
 	let mut properties = Properties::new();
 	properties.insert("tokenDecimals".into(), 9.into());
-	properties.insert("tokenSymbol".into(), "ANMO".into());
+	properties.insert("tokenSymbol".into(), "ANML".into());
 	properties.insert("ss58Format".into(), 64.into());
```
Comment (nblogist) [inline node/src/chain_spec.rs]: @justinFrevert plas review [this](https://www.notion.so/dot-matrix/ChainSpec-272b336fbe064820b7c064f2daf9f290) ticket

Comment (nblogist) [reply node/src/chain_spec.rs]: @justinFrevert  for reference please see [here](https://github.com/hicommonwealth/edgeware-node/blob/master/node/cli/src/chain_spec.rs#L541-L544)

**Review by aelesbao — DISMISSED** [review]

**Review by aelesbao — DISMISSED** [review]

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -152,7 +152,7 @@ parameter_types! {
 		::with_sensible_defaults(2 * WEIGHT_PER_SECOND, NORMAL_DISPATCH_RATIO);
 	pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
 		::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
-	pub const SS58Prefix: u8 = 92;
+	pub const SS58Prefix: u8 = 182;
```
Comment (nblogist) [inline runtime/src/lib.rs:155]: @justinFrevert typo?

Comment (justinfrevert) [reply runtime/src/lib.rs:155]: Yes

**Comment by justinfrevert** [comment]
Needs more work. Will need to properly set up the node with another validator.

**Pull request closed without merging**